### PR TITLE
graphnode v 0.0.6

### PR DESCRIPTION
**Pull request by orenyodfat**
graph-cli v 0.8.0
graph-ts v 0.6.0

Add redeemer to subgraph

### CODE REVIEW

File: ops/DummyMapping/datasource.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+entities:
+  - Dummy
```
Comment (ben-kaufman) [inline ops/DummyMapping/datasource.yaml]: What's this?

File: ops/DummyMapping/mapping.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+import 'allocator/arena';
```
Comment (ben-kaufman) [inline ops/DummyMapping/mapping.ts]: What's this?

File: ops/generate-subgraph.js [hunk 3]
```diff
@@ -13,7 +13,22 @@ async function generateSubgraph () {
 
   const dataSources = mappings.map(mapping => {
     var contract = mapping.name
-    const { abis, entities, eventHandlers } = yaml.safeLoad(fs.readFileSync('src/mappings/' + mapping.mapping + '/datasource.yaml', 'utf-8'))
+    var abis
```
Comment (ben-kaufman) [inline ops/generate-subgraph.js]: What are those Dummy stuff? Also please declare the vars in a single statement.

File: test/integration/domain.spec.ts [hunk 4]
```diff
@@ -264,34 +264,45 @@ describe('Domain Layer', () => {
...
+
+    expect(dao.schemes).toContainEqual(
+      {
           address: addresses.ContributionReward.toLowerCase(),
           dao: {
             id: addresses.Avatar.toLowerCase(),
           },
         },
-        {
+    );
```
Comment (ben-kaufman) [inline test/integration/domain.spec.ts]: Why so many new expect statements?

File: ops/DummyMapping/mapping.ts (hunk 2 above)
Comment (ben-kaufman) [reply ops/DummyMapping/mapping.ts]: Also as far as I remember this statement became unnecessary a few versions ago...

File: test/integration/domain.spec.ts (hunk 4 above)
Comment (orenyodfat) [reply test/integration/domain.spec.ts]: to ignore the order of the array entries.

**Pull request merged**